src/elements/IconButton/IconButton.story.tsx

```diff
@@ -0,0 +1,83 @@
+import { Fragment } from 'react';
+import { IconButton } from './IconButton';
+
+export default {
+  title: 'Components/Elements/IconButton',
+  component: IconButton
+};
+
+const BellIcon = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="16"
+    height="16"
+    fill="white"
+    viewBox="0 0 16 16"
+  >
+    <path d="M8 16a2 2 0 0 0 2-2H6a2 2 0 0 0 2 2zm.995-14.901a1 1 0 1 0-1.99 0A5.002 5.002 0 0 0 3 6c0 1.098-.5 6-2 7h14c-1.5-1-2-5.902-2-7 0-2.42-1.72-4.44-4.005-4.901z" />
+  </svg>
+);
+
+export const IconAndTextSizes = () => (
+  <Fragment>
+    <IconButton size="small" icon={<BellIcon />}>
+      Small
+    </IconButton>
+    {` `}
+    <IconButton size="medium" icon={<BellIcon />}>
+      Medium
+    </IconButton>
+    {` `}
+    <IconButton size="large" icon={<BellIcon />}>
+      Large
+    </IconButton>
+  </Fragment>
+);
+
+export const IconAndTextPositions = () => (
+  <Fragment>
+    <IconButton size="medium" icon={<BellIcon />} direction="row">
```

Comment:
I dno't love mixing a stack in w/ the button. 

I looked at some other libraries how they did it:
- https://mui.com/joy-ui/react-button/
- https://mui.com/material-ui/react-button/#buttons-with-icons-and-label

I think it makes sense to have the API something like those:

Only an icon would be used in a `IconButton` like below, also moved the `icon` to the `children`:

```
<IconButton>
   <MyIcon />
</IconButton>
```

and then follow the pattern like we did for `Input` similar to how MUI does it for a Icon + Button.

Reply:
Added `startIcon` and `endIcon` props to the `Button` component and refactored the `IconButton` component to accept the `icon` as `children`.


https://github.com/reaviz/reablocks/assets/33908100/fa6c8a74-e640-434b-9357-dabfbfcd175f